AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/CampaignAdminControllerTests.cs

```diff
@@ -23,21 +23,40 @@ public class CampaignAdminControllerTests : InMemoryContextTest
         //delete this line when all unit tests using it have been completed
         private readonly Task taskFromResultZero = Task.FromResult(0);
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact()]
         public void IndexSendsCampaignListQueryWithCorrectDataWhenUserIsOrgAdmin()
         {
+            Organization aginAware = AgincourtAwarenessExtended(Context);
+            Organization pwPrepares = ParkwoodsPreparesExtended(Context);
+            Context.SaveChanges();
+
+            // Org admin should only see own campaigns
+            CampaignController controller = CampaignControllerWithListQuery(Context, UserType.OrgAdmin.ToString(), aginAware.Id);
+            ViewResult view = (ViewResult)controller.Index();
+            IEnumerable<AllReady.Areas.Admin.Models.CampaignSummaryModel> viewModel = (IEnumerable<AllReady.Areas.Admin.Models.CampaignSummaryModel>)view.ViewData.Model;
```

Comment:
you can delete the fully qualified `AllReady.Areas.Admin.Models.` namespace form the font of CampaignSummaryModel